pkgs/servers/clickhouse/default.nix

```
@@ -178,6 +178,7 @@ in mkDerivation rec {
     "-DENABLE_TESTS=OFF"
     "-DCOMPILER_CACHE=disabled"
     "-DENABLE_EMBEDDED_COMPILER=ON"
+    "-DNO_ARMV81_OR_HIGHER=1"
```

```suggestion
    "-DNO_ARMV81_OR_HIGHER=1"
```

Use `stdenv.hostPlatform.isLinux && stdenv.hostPlatform.isAarch64` with lib.optionals here.

---

Done